packages/api/src/dwn-api.ts

```diff
@@ -353,7 +356,7 @@ export class DwnApi {
           agentResponse = await this.agent.processDwnRequest(agentRequest);
         }
 
-        const { reply: { entries, status } } = agentResponse;
+        const { reply: { entries, status, paginationMessageCid } } = agentResponse;
```

Comment:
@LiranCohen In the hopes of providing the most intuitive developer experience, I was thinking that perhaps we name `paginationMessageCid` something that alludes to this value being the pagination cursor.

I was reading through the PR that originally introduced the pagination feature.  I can understand the reasons for using the `messageCid` as the token but it seems like providing a more descriptive name to the value returned would have been more intuitive for developers using the feature.  Particularly since so many other pagination implementations often return to the value as a `token` or `cursor`.

Thoughts?

Reply:
@frankhinek 

I agree I think what we landed on wrt the cursor objects is less than optimal, especially looking at it again after some time.

I'm leaning towards a more generic naming, and we can define the explicitness of the values within docs/spec/comments.

Example:
```javascript
// Query pagination object in descriptor
export type Pagination = {
  cursor?: string
  limit?: number
};

export type RecordsQueryResponse = {
  status: UnionMessageReply['status'];
  records?: Record[];
  cursor?: string;
};
```

If we really want to be explicit about the cursor being a `messageCid`, which could have some value, maybe something like `cursorMessageCid`?

Reply:
@frankhinek Originally was thinking of making the change here first to get it in, but then realized that it would be a hassle since the pagination object with `messageCid` is in the descriptor of the request.

I opened a PR in `dwn-sdk-js` to get this in and bubble it up to here.